lib/src/git.rs

```diff
@@ -326,7 +328,10 @@ pub fn import_some_refs(
     } else {
         vec![]
     };
-    let stats = GitImportStats { abandoned_commits };
+    let stats = GitImportStats {
+        abandoned_commits,
+        new_remote_refs: BTreeSet::from_iter(changed_remote_refs.keys().cloned()),
```

Comment:
simply return the `changed_remote_refs` map? The caller would want to know if the ref is created `(absent, present)` or deleted `(present, absent)`.

Reply:
resolved